setup.py

```diff
@@ -45,30 +44,24 @@
     'processing where possible.')
 
 setup(
-    name='turbinia',
-    version=turbinia.__version__,
+    name='turbinia', version=turbinia.__version__,
```

Comment:
Did these changes come from YAPF?  If so, maybe we should exclude this file from YAPF by adding a comment?